### PR TITLE
Add Node API documentation

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: validators/node-api.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+## Public/private APIs
+
+Methods that are marked as `ALLOWED` could be safely used by validators directly or by their custom tools.  
+Usage examples are available for public APIs.
+
+Direct use:
+
+- `ALLOWED` - API is safe to use directly by custom tools
+- `NOT ALLOWED` - API shouldn't be used directly
```
Comment (badrogger) [inline validators/node-api.md]: May be it's better to hide such endpoints. What the point to show them in docs?

File: validators/node-api.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      - [Methods](#methods-3)
+      - [Usage](#usage-3)
+    + [SSL](#ssl)
+      - [Methods](#methods-4)
+      - [Usage](#usage-4)
+    + [Wallet](#wallet)
+      - [Methods](#methods-5)
+      - [Usage](#usage-5)
+
+## API structure
```
Comment (badrogger) [inline validators/node-api.md]: I think _Endpoint structure_ is a little more accurate.

File: validators/node-api.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+```bash
+/api/{API_VERSION}/{BLUEPRINT_NAME}/{METHOD_NAME}
```
Comment (badrogger) [inline validators/node-api.md]: Not sure that blueprint is standard term. May be we should rename to group ?

File: validators/node-api.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+|-------------------------------------|--------|---------------|----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|
+| `/api/v1/node/info`                 | `GET`  | `ALLOWED`     | Get node info                                                                                | N/A                                                                                                                                                      | N/A                                      |
+| `/api/v1/node/register`             | `POST` | `NOT ALLOWED` | Register SKALE node on contracts                                                             | `ip` - internal IP address of the node, `public_ip` - public IP for RPC connections & consensus, `port` - base port for node sChains, `name` - node name | `gas_price`, `gas_limit`, `skip_dry_run` |
```
Comment (badrogger) [inline validators/node-api.md]: Let's hide public ip. It's not used anywhere and may confuse validators.

**Review by badrogger — CHANGES_REQUESTED** [review]

File: validators/node-api.md (hunk 1 above)
Comment (dmytrotkk) [reply validators/node-api.md]: This separation is nominal tbh. Maybe it's better to rename `NOT ALLOWED` to `NOT RECOMMENDED` or something like this. Cause actually they could be called directly with caution.

File: validators/node-api.md (hunk 2 above)
Comment (dmytrotkk) [reply validators/node-api.md]: 👍

File: validators/node-api.md (hunk 3 above)
Comment (dmytrotkk) [reply validators/node-api.md]: renamed

**Review by badrogger — DISMISSED** [review]

**Pull request closed without merging**